image-resize/function.js

```diff
@@ -0,0 +1,59 @@
+const cp = require('child_process');
+const request = require('request-promise-native');
+const file = require('./file');
+
+exports.handler = async (body, context) => {
+
+  // Get an image URL from the request body or query
+  // string (useful for invoking directly from a browser)
+  const uri = body.url || context.request.query.url;
+  if (!uri) {
```

Comment:
This is not a good check to have. Instead explicitly check for the `undefined` case. Javascript truthy semantics are not like C truthy semantics.